<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerCandidati</class>
 <widget class="QMainWindow" name="ManagerCandidati">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManagerCandidati</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cognome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data di Nascita</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Genere</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Occupazione</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Idoneit√†</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="modifica_btn">
        <property name="text">
         <string>Modifica</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rimuovi_btn">
        <property name="text">
         <string>Rimuovi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nuovoCandidato_btn">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nuovo Candidato</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="caricaFile_btn">
        <property name="text">
         <string>Carica file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="grafico_btn">
        <property name="text">
         <string>Mostra Grafico</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
